### ISSUE TITLE
Cannot easily drill down the collapsed subprocess as append element gets focussed 

### ISSUE THREAD

**Issue by GeethaParthasarathy** [camunda/camunda-modeler#4548]
### Describe the bug

When we have a collapsed subprocess with the new context pad hovering on the drill down arrow also focusses on the append element making it difficult click on the drill down arrow

### Steps to reproduce

1. Create a collapsed subprocess
2. Try to click on the drill down arrow

https://github.com/user-attachments/assets/ec514acb-eeeb-42db-bf1e-21910af92bcf

### Expected behavior

Focus on the drill down arrow and make it possible to click 

### Environment

- OS: [MacOS 10.2]
- Camunda Modeler Version: [5.28.0]
- Execution Platform: [Camunda 8]
- Installed plug-ins: [...]

**Comment by nikku** [comment]
@philippfromme @lmbateman I'd be interested to understand your assessment of this UX challenge. From what the looks of it drilldown is not easily possible using the new context pad.

**Comment by lmbateman** [comment]
I thought from Geetha's example that there might be some steps we could take to fix it, like separating the icons a bit more and perhaps decreasing the click target for the append menu, but now I think it will require a bit more than that:
<img width="351" alt="image" src="https://github.com/user-attachments/assets/67f481b0-98a4-4501-9d6b-4d4403e5c1b8">

**Comment by nikku** [comment]
@philippfromme Maybe you can work together with @lmbateman here?

**Comment by philippfromme** [comment]
I thought about moving it so it's inside the element similar to DMN decision table and literal expression icons:

<img alt="brave_6Cio3DVeTQ" src="https://github.com/user-attachments/assets/623bc1ce-5eae-4241-af96-a7d728b81db9">

However, if there's a boundary event it's not ideal:

<img alt="brave_LjcTZyeSdL" src="https://github.com/user-attachments/assets/cdc3f0a0-d856-4a02-b3d7-58d0fff344ab">

I also tried moving the overlay slightly will still conflict with the context pad when zoomed in since the overlay scales and the context pad doesn't.

<img width="508" alt="brave_wbk334Xf2D" src="https://github.com/user-attachments/assets/4a1dfb1e-11e6-44a8-8754-e082fd3cf82e">

Not scaling it will make it look detached when zoomed in.

![Uploading brave_U5AGBQTrts.png…]()


Moving the button to the context pad doesn't make sense because the viewer doesn't have a context pad.

**Comment by philippfromme** [comment]
@leah Wondering if any of the options you'd approve?

**Comment by lmbateman** [comment]
I like the idea of moving it into the element, but two thoughts:
- It seems like the collapsed subprocess is mutually exclusive with other task types. In that case, I wonder about putting the icon in the upper left, like the other task type icons. One downside is that this would be the only interactive icon in that position in the BPMN editor, but there is a precedent in the DMN editor.
- There are a lot of blue and white plus signs floating around. Is the plus-sign-inside-a-square part of the BPMN standard? If so, I would keep the separate blue and white arrow and go with the option of moving the overlay slightly (the option that requires more work, sadly).

**Comment by philippfromme** [comment]
I agree that moving the icon to the upper left corner would be cofusing.

The BPMN spec states: 

> The Sub-Process marker MUST be a small square with a plus sign (+) inside. The square MUST be
 positioned at the bottom center of the shape.

So we cannot change that.

So it seems like we'll keep the general position bottom right of the element but adjust it slightly and make sure it behaves like the context pad in terms of position and scaling when zooming.

**Comment by barmac** [comment]
I don't have a strong opinion, but if we are to change anything here, I'd like to compare the experience on the default zoom level too:
<img width="1599" alt="image" src="https://github.com/user-attachments/assets/bd94c281-0dd2-4feb-be23-e6df69314780">

**Comment by philippfromme** [comment]
At the default zoom level only the position would change compared to now:

![image](https://github.com/user-attachments/assets/1dbe6910-b2c9-43fb-84d4-710a123c9a49)

**Comment by barmac** [comment]
This got reported [again](https://camunda.slack.com/archives/C0693F1NFK5/p1744659288251669) and [again](https://camunda.slack.com/archives/C0693F1NFK5/p1745522076248129).

@lmbateman do you plan to prepare a proposal for this?

**Comment by philippfromme** [comment]
If we simplify the context pad and remove the quick attach we can get away with moving the overlay just slighty.

![Image](https://github.com/user-attachments/assets/95325531-07cb-411d-87b4-b6668eaebec4)

**Comment by lmbateman** [comment]
Here's [the latest proposal](https://www.figma.com/design/OqQVvoFXqANDj7LZWt1BAq/Enhancing-Information-Accessibility-and-Visibility-on-the-Canvas?node-id=563-1426&t=DXtiq7gFtILj0Cak-4), with a couple of ideas for how to hide the comment and documentation icons, but without Philipp's idea of removing the quick attach.
Option 1: Error icons at top of element; toggle to turn icons on and off
Option 2: Error icons at bottom of element (current position); canvas tool to turn icons on and off

**Comment by nikku** [comment]
> Option 1: Error icons at top of element; toggle to turn icons on and off

I appologize the delay. My preferred option was, and continues to be option 1. The toggle is "opening the problems panel" in Desktop Modeler, it could very well also be an additional trigger (where we see the need).

**Comment by nikku** [comment]
@AlekseyManetov Fixed with the latest release?

**Comment by AlekseyManetov** [comment]
> [@AlekseyManetov](https://github.com/AlekseyManetov) Fixed with the latest release?

Yes, this is fixed with the new append pad implementation

<img width="809" height="284" alt="Image" src="https://github.com/user-attachments/assets/9a79e421-73fb-4ecf-9c25-50580490b141" />
